herbert_core/classes/MPIFramework/@ClusterParpoolWrapper/ClusterParpoolWrapper.m

```diff
@@ -114,6 +114,18 @@
             
             cl  = parcluster();
             
+            % By default Matlab only utilises physical cores; enable use of
+            % logical cores if required
+            if cl.NumWorkers < obj.n_workers
+                [n_physical_cores, n_logical_cores] = get_num_cores();
+                if obj.n_workers <= n_logical_cores
+                    cl.NumWorkers = obj.n_workers;
+                    if cl.NumWorkers > n_physical_cores
```

Comment:
should these two steps be the other way around? 
```
if (obj.n_workers > n_physical_cores) 
    disp("...enabling hyperthreading...")
end
cl.NumWorkers = obj_n_workers
```

Reply:
Really, the print statement isn't necessary, I put it in there as a debugging statement originally. I've removed it for now, but it can be put back in